wp-content/plugins/core/src/Blocks/Blocks_Definer.php

```diff
@@ -20,6 +22,7 @@ public function define(): array {
 			] ),
 
 			self::STYLES          => DI\add( [
+				// DI\get( Button::class ),
```

Comment:
@dpellenwood I'm sure I'm doing something wrong w/ namespaces but I couldn't seem to get my custom styles for the Button block to register. Can I get a hand w/ this when you have a min?

Reply:
@jamiepaul , this should be fixed now. You may have to run `composer dump-autoload` after pulling the latest into your local branch.